src/schema/imported/manifest.json

```diff
@@ -780,6 +780,16 @@
           "default": "*",
           "description": "Maximum version of Gecko to support. Defaults to '*'. (Requires Gecko 45)",
           "pattern": "^[0-9]{1,3}(\\.[a-z0-9*]+)+$"
+        },
+        "install_origins": {
+          "type": "array",
+          "optional": true,
+          "minItems": 1,
+          "maxItems": 5,
+          "items": {
+            "type": "string",
+            "format": "origin"
+          }
```

Comment:
As a side note, as part of the review on the Firefox patch, I did mention that `FirefoxSpecificProperties` JSONSchema doesn't allow arbitrary properties that are not explicitly listed in its schema, and so a manifest.json file including `install_origins` inside `browser_specific_settings.gecko` (or `applications.gecko`) would be invalid for older Firefox releases (summarized from [this comment](https://phabricator.services.mozilla.com/D129105#inline-715990)).

I briefly touched base about that with Shane yesterday, and we agreed to discuss about it more today during out team standup.

And so, for now there are no changes requested, but we may have to change this to match what we will agree on doing on the Firefox side (e.g. we can't exclude yet that we may choose to move the property outside of `FirefoxSpecificProperties`, e.g. in a new top level manifest key where the pre-existing JSONSchema does already allow additional properties).

Reply:
> TIL. We want developers to adopt `install_origins` sooner rather than later, if it makes their add-on incompatible with older Firefox versions that sounds like a significant problem...

It is definitely a significant problem (and I'm really glad that we spotted it earlier in the review passes).
That's basically why yesterday I asked Shane to have a quick preliminary chat about that single point from my review pass on the Firefox patch, and we agreed to discuss about it more during the team standup today.

I'm confident we will get to an agreement quickly, either today or in the worst case by the end of this week (you are also welcome to join us if you want to, I can ping you when we are about to discuss about this point, but in any case I will report back here the outcome of that discussion).

Reply:
@diox we discussed about this issue during the meeting (and Shane told me that it has been discussed also in another meeting, I think it was the Ops one) and we agreed to move the key to be a top level manifest key (same name and same JSONSchema definition, just moved to be a top level one).

I'm not yet sure if it is meant to be on the `ManifestBase` one, but it seems a reasonable assumption given that it was in the `FirefoxSpecificProperties`, which are part of the `ManifestBase` schema.